packages/@okta/vuepress-site/docs/guides/event-hook-implementation/enable-and-verify/index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Enable and verify Event Hook
+---
+The Event Hook must be set up and verified within your Okta Developer Console.
+
+### Set up the Event Hook
+To set up the Event Hook:
+
+1. Navigate to the **Workflow** > **Event Hooks** page.
+
+2. Click **Create Event Hook**.
+
+3. Add a name for the hook in the **Add Event Hook Endpoint** dialog box (in this example, "Deactivated User Event Hook").
+
+4. Add your external service URL, including endpoint. For example, use your Glitch project name with the endpoint: `https://your-glitch-projectname.glitch.me/userDeactivated`.
+
+5. Include authentication field and secret. In this example:
+
+    - **Authentication field** = `authorization`
+
+    - **Authentication secret** = `Basic YWRtaW46c3VwZXJzZWNyZXQ=`
+
+6. In the **Requests** section of the dialog box, subscribe to the Event Type you want to monitor. In this example, a user deactivated in the Okta org: `User deactivated`
+
+7. Click **Save & Continue**.
+
+8. You can complete the one-time verification Okta call at this time or verify the Event Hook later. If using the Glitch example, proceed to verification.
+
+> **Note** You can also set up an Event Hook using an API. See [Inline Hooks Management](/docs/reference/api/inline-hooks/) for further information.
```

Comment:
I think we want to link to [Event Hooks Management API](http://localhost:8080/docs/reference/api/event-hooks/)  here instead of Inline Hooks Management.